docs/source/man.rst

```diff
@@ -388,6 +388,48 @@ The ``qpc scan cancel`` command cancels the execution of a scan. A canceled scan
   Required. Contains the identifier of the scan to cancel.
 
 
+Reports
+--------
+
+Use the ``qpc report`` command to view the reports from a scan.  Reports can be viewed as JSON or a CSV.  There are two different types of report: detail and summary.
+
+
+View Detail Report
+~~~~~~~~~~~~~~~~~~~
+The ``qpc report detail`` command provides unprocessed facts gathered during a scan.  These are the raw output from network, VCenter, and Satellite scans.
```

Comment:
did we uppercase VCenter and Satellite elsewhere?  at minimum I think its vCenter.

Reply:
all lower now.